Eradication/VectorToHumanAdapter.cpp

```diff
@@ -3,16 +3,25 @@
 
 #include "VectorToHumanAdapter.h"
 #include "INodeContext.h"
+#include "SimulationEnums.h"
 
 namespace Kernel
 {
     BEGIN_QUERY_INTERFACE_BODY( VectorToHumanAdapter )
         HANDLE_INTERFACE( IIndividualHumanEventContext )
     END_QUERY_INTERFACE_BODY( VectorToHumanAdapter )
 
-    VectorToHumanAdapter::VectorToHumanAdapter( INodeContext* pNodeContext, uint32_t vectorID )
+    VectorToHumanAdapter::VectorToHumanAdapter(INodeContext* pNodeContext, uint32_t vectorID)
+        : m_pNodeContext(pNodeContext)
+        , m_VectorID(vectorID)
+        , m_VectorGender(VectorGender::VECTOR_FEMALE)
```

Comment:
You can change this to:

```
 VectorToHumanAdapter::VectorToHumanAdapter(INodeContext* pNodeContext, uint32_t vectorID)
    : ectorToHumanAdapter( pNodeContext, vectorID, VectorGender::VECTOR_FEMALE )
```

This is relatively new to C++ and I'm still trying to remind myself that you can do it.